pages/[...slug].vue

```diff
@@ -18,6 +18,39 @@
   <EpPageMeta :doc="doc" />
   <NuxtLayout>
     <div class="px-4 print:px-8 mt-10 mb-96 print:mb-0">
+      <div>
+        <nav>
+          <span v-if="breadcrumbs.length != 1">
+            <NuxtLink :to="'/'">
+              <Icon name="heroicons:home-20-solid" size="2em" />
+            </NuxtLink>
+            {{ '>' }}
+          </span>
+
+          <span
+            v-for="x in breadcrumbs.slice(1, breadcrumbs.length - 1)"
```

Comment:
Give appropriate names to items in iteration.

```suggestion
            v-for="crumb in breadcrumbs.slice(1, breadcrumbs.length - 1)"
```